src/test/java/eu/tsystems/mms/tic/testerra/plugins/selenoid/AbstractSelenoidTest.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Testerra
+ *
+ * (C) 2021, Mike Reiche,  T-Systems Multimedia Solutions GmbH, Deutsche Telekom AG
+ *
+ * Deutsche Telekom AG and all other contributors /
+ * copyright owners license this file to you under the Apache
+ * License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package eu.tsystems.mms.tic.testerra.plugins.selenoid;
+
+import eu.tsystems.mms.tic.testframework.constants.Browsers;
+import eu.tsystems.mms.tic.testframework.report.model.context.ClassContext;
+import eu.tsystems.mms.tic.testframework.report.model.context.MethodContext;
+import eu.tsystems.mms.tic.testframework.report.model.context.SessionContext;
+import eu.tsystems.mms.tic.testframework.report.model.context.SuiteContext;
+import eu.tsystems.mms.tic.testframework.report.model.context.TestContext;
+import eu.tsystems.mms.tic.testframework.report.utils.ExecutionContextController;
+import eu.tsystems.mms.tic.testframework.testing.TesterraTest;
+import eu.tsystems.mms.tic.testframework.useragents.ChromeConfig;
+import eu.tsystems.mms.tic.testframework.useragents.FirefoxConfig;
+import eu.tsystems.mms.tic.testframework.webdrivermanager.WebDriverManager;
+import eu.tsystems.mms.tic.testframework.webdrivermanager.WebDriverProxyUtils;
+import java.util.Optional;
+import org.openqa.selenium.Proxy;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.firefox.FirefoxOptions;
+import org.testng.Assert;
+import org.testng.ITestResult;
+import org.testng.annotations.BeforeSuite;
+
+public abstract class AbstractSelenoidTest extends TesterraTest {
+
+    @BeforeSuite
+    public void configureChromeOptions() {
+        Proxy proxy = new WebDriverProxyUtils().getDefaultHttpProxy();
+
+        WebDriverManager.setUserAgentConfig(Browsers.chrome, new ChromeConfig() {
+            @Override
+            public void configure(ChromeOptions options) {
+                options.setProxy(proxy);
+            }
+        });
+
+        WebDriverManager.setUserAgentConfig(Browsers.firefox, new FirefoxConfig() {
+            @Override
+            public void configure(FirefoxOptions firefoxOptions) {
+                firefoxOptions.setProxy(proxy);
+            }
+        });
+
+    }
+
+    protected boolean Video_is_present_in_SessionContext(String methodName, boolean expectPresence) {
```

Comment:
This should be camel case starting with lower character, should'nt it?
Or did i miss some new naming rules?

Reply:
The naming rules allow these snake_case methods in tests, which it basically is.